app/services/VatCardBuilderService.scala

```diff
@@ -41,11 +42,15 @@ class VatCardBuilderServiceImpl @Inject()(val messagesApi: MessagesApi,
                                           linkProviderService: LinkProviderService
                                          )(implicit ec: ExecutionContext) extends VatCardBuilderService {
 
+  val today = LocalDate.now()
```

Comment:
This needs to be def, otherwise the instance of the class will keep "today" frozen at whenever it was created

Reply:
Fixed